twisted/test/test_compat.py

```diff
@@ -315,6 +317,26 @@ def test_python3(self):
 
 
 
+class PYPYTest(unittest.SynchronousTestCase):
+    """
+    Identification of PyPy.
+    """
+
+    def test_pypy(self):
+        """
+        On PyPy, C{_PYPY} is True.
```

Comment:
Sure!  i was trying to remain consistent with `PY3Tests`.  Maybe a new PR for that?